pkg/vpcmodel/explainabilityPrint.go

```diff
@@ -204,37 +204,37 @@ func (g *groupedConnLine) explainPerCaseStr(c *VPCConfig, src, dst EndpointElem,
 		return fmt.Sprintf("%v\tThe dst is external but there is no Floating IP or Public Gateway connecting to public internet\n",
 			noConnection)
 	case ingressBlocking || egressBlocking || loadBalancerBlocking:
-		return fmt.Sprintf("%vconnection is blocked %s"+tripleNLVars, noConnection,
-			blockedByString(ingressBlocking, egressBlocking, loadBalancerBlocking),
+		return fmt.Sprintf("%v%s"+tripleNLVars, noConnection,
+			blockSummery(ingressBlocking, egressBlocking, loadBalancerBlocking),
 			headerPlusPath, details)
 	default: // there is a connection
 		return existingConnectionStr(c, connQuery, src, dst, conn, path, details)
 	}
 }
 
-// blockedByString() return the block string according to the flags
-func blockedByString(ingressBlocking, egressBlocking, loadBalancerBlocking bool) string {
+// blockSummery() return a summery of the rules that block the connection, for example:
+// "connection is blocked both by ingress and egress, and can not be initiated by Load Balancer"
+func blockSummery(ingressBlocking, egressBlocking, loadBalancerBlocking bool) string {
```

Comment:
```suggestion
// blockSummary() return a summary of the rules that block the connection, for example:
// "connection is blocked both by ingress and egress, and cannot be initiated by Load Balancer"
func blockSummary(ingressBlocking, egressBlocking, loadBalancerBlocking bool) string {
```

try checking typo checks plugin for vscode

Reply:
fixed